pytest.ini

```diff
@@ -1,7 +1,7 @@
 [pytest]
 addopts = --doctest-modules
 doctest_optionflags = NORMALIZE_WHITESPACE ELLIPSIS
-norecursedirs = tests/_test_local tests/test_cli tests/_test_data tests/full_test_notebooks tidy3d/web
+norecursedirs = tests/_test_local tests/test_cli tests/_test_data tests/_test_notebooks tidy3d/web
```

Comment:
I thought if the folder starts with `_` it will already ignore it?

Reply:
So I modified the default behavior via the `pytest.ini`, so whenever the pytest is run from the frontend it will use this configuration to perform even the docstring tests which was reported not to properly work before this check.